### PR TITLE
DOCSP-28563 Adds telemetry option and setting

### PR DESCRIPTION
**Pull request by kennethdyer**
## Description

Adds the `--disableTelemetry` option and setting for `mongosync`.

## Staging

* [`--disableTelemtry` option](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-28563-telemetry-toggle/reference/mongosync/#std-option-mongosync.--disableTelemetry)

* [`disableTelemetry` setting](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-28563-telemetry-toggle/reference/configuration/#mongodb-setting-disableTelemetry)

## Jira

[DOCSP-28563](https://jira.mongodb.org/browse/DOCSP-28563)

## Build
* [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64307673b49106a498392f36)

### CODE REVIEW

File: source/includes/opts/disableTelemetry.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+Disables the collection and sharing of ``mongosync`` telemetry data with MongoDB.
```
Comment (mdb-ashley) [inline source/includes/opts/disableTelemetry.rst:2]: [praise] Nice work making this an include!

**Review by mdb-ashley — APPROVED** [review]
This LGTM, Kenneth! Nice work.

**Comment by kennethdyer** [comment]
Marking this one as blocked, per Ajay Vijaykumar.  They don't want to ship it until the backing feature is ready (to happen sometime after 1.3.0).

File: source/includes/opts/disableTelemetry.rst (hunk 1 above)
Comment (ajayvijayakumar123) [inline source/includes/opts/disableTelemetry.rst:2]: Here's a link to Segment: https://segment.com/
```suggestion
Disables the sharing of ``mongosync`` telemetry data with Segment. 
```

File: source/includes/opts/disableTelemetry.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (ajayvijayakumar123) [inline source/includes/opts/disableTelemetry.rst:1]: Could we also include on the docs that we will **not** be tracking IP addresses, hostnames, usernames, passwords, credentials, data stored in users' MongoDB deployments, or personable identifiable information when sending telemetry to Segment?

**Review by ajayvijayakumar123 — CHANGES_REQUESTED** [review]
Left a couple of suggestions. Thanks!

**Comment by davemungo** [comment]
This was incorporated into
 
DOCSP-29499 [REP] Add Telemetry to mongosync
https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/128

**Pull request closed without merging**